test/unit/findWealthAccounts.spec.ts

```diff
@@ -0,0 +1,56 @@
+import { expect, use } from 'chai';
+import { utils, type providers } from 'ethers';
+import {
+  REGTEST_CHAIN_ID,
+  findWealthyAccount,
+} from 'src/commands/findWealthyAccount';
+import chaiAsPromised from 'chai-as-promised';
+
+use(chaiAsPromised);
+
+const TESTNET_CHAIN_ID = 31;
+const MAINNET_CHAIN_ID = 30;
+
+describe('findWealthAccounts', function () {
+  const expectedSigner = {
+    getAddress: () => '0x123abc',
+    getBalance: () => Promise.resolve(utils.parseUnits('2', 'ether')),
+  };
+
+  const getMockedProvider = (expectedChainId: number) =>
+    ({
+      getNetwork: () => ({ chainId: expectedChainId }),
+      listAccounts: () => [1, 2, 3],
+      getSigner: () => expectedSigner,
+    } as unknown as providers.JsonRpcProvider);
+
+  const expectFindWealthyAccountToFail = async (chainId: number) => {
+    const mockRpcProvider = getMockedProvider(chainId);
+    await expect(findWealthyAccount(mockRpcProvider)).to.rejectedWith(
+      'Unlocked accounts are allowed for testing purposes only'
+    );
+  };
+
+  it('should not raise an error if it is used for dev purposes', async function () {
```

Comment:
what about the if the wealthy account is found, I think that we should test it aswell.

Reply:
@franciscotobar  Isn't this scenario tested here? At line 37, I check if the found account is the expected one. Do you mean anything else?

Reply:
Yeah I think so :S